configuration/pih/petl/jobs/general/visit/source.sql

```diff
@@ -0,0 +1,53 @@
+select encounter_type_id into @tb_general from encounter_type where uuid= 'aa42cc6c-b9ee-4850-926c-dda4bb14d890';/*50 tb intake*/
```

Comment:
How does this differ from the diagnoses source one?

Reply:
It's the same encounter type, but the way COR works is that patient comes to the clinic to get a diagnosis if they have a disease or not (covid, hiv, tb). So it counts as a visit.